src/praisonai-bot/praisonai_bot/bots/whatsapp.py

```
@@ -898,6 +902,83 @@ async def _post_chunk(chunk_text: str = chunk) -> dict:
+                        raise RuntimeError(
+                            f"WhatsApp interactive send error: {result['error']}"
+                        )
+                    return result
+
+            result = await self.deliver_outbound(
+                _post,
+                channel_id=target,
+                reply_text=rendered.get("text") or "",
+            )
+            return result.get("messages", [{}])[0].get("id", "") or None
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)